.github/workflows/publish.yml

```diff
@@ -0,0 +1,15 @@
+name: Publish
+
+on:
+  push:
+    tags: ["*"] # Releases
```

Comment:
We should also publish snapshots for `branches´ like I described in the play-slick pr.